mti_nma/bin/all.py

```diff
@@ -63,37 +66,50 @@ def run(
             A debug flag for the developer to use to manipulate how much data runs,
             how it is processed, etc.
             Default: False (Do not debug)
-        cell_flag: bool
-            Flag for wether to include cell membrane in analysis. The nucleus is always
-            analyzed, and this flag allow you to either additionally analyze the cell
-            membrane (True) or not (False).
+        structs: List
+            List of structure data to run pipeline on. Currently, only
+            'Nuc' (nuclear membrane) and 'Cell' (cell membrane) are supported.
+
         Notes
         -----
         Documentation on prefect:
         https://docs.prefect.io/core/
         Basic prefect example:
         https://docs.prefect.io/core/
         """
+
         # Initalize steps
-        singlecell = steps.Singlecell()
-        shparam = steps.Shparam()
-        avgshape = steps.Avgshape()
-        nma = steps.Nma()
+        if "Nuc" in structs:
+            single_nuc = steps.Singlecell(step_name="single_nuc")
+            shparam_nuc = steps.Shparam(step_name="shparam_nuc")
+            avgshape_nuc = steps.Avgshape(step_name="avgshape_nuc")
+            nma_nuc = steps.Nma(step_name="nma_nuc")
+
+        if "Cell" in structs:
+            single_cell = steps.Singlecell(step_name="single_cell")
+            shparam_cell = steps.Shparam(step_name="shparam_cell")
+            avgshape_cell = steps.Avgshape(step_name="avgshape_cell")
+            nma_cell = steps.Nma(step_name="nma_cell")
+
+        if "Nuc" in structs and "Cell" in structs:
+            compare_nuc_cell = steps.CompareNucCell()
 
         # Choose executor
         if debug:
             exe = LocalExecutor()
             distributed_executor_address = None
+            log.info(f"Debug flagged. Will use threads instead of Dask.")
         else:
             if distributed:
                 # Create or get log dir
                 # Do not include ms
                 log_dir_name = datetime.now().isoformat().split(".")[0]
-                log_dir = Path(f"~/.dask_logs/mti_nma/{log_dir_name}")
+                log_dir = Path(f"~/.dask_logs/mti_nma/{log_dir_name}").expanduser()
```

Comment:
Yep this `.expanduser()` solves the `~` directory from being made. So can remove the gitignored line.